src/ONYXKEYS.js

```diff
@@ -76,6 +76,8 @@ export default {
     // Contains the users's block expiration (if they have one)
     NVP_BLOCKED_FROM_CONCIERGE: 'private_blockedFromConcierge',
 
+    NVP_PRIVATE_PUSH_NOTIFICATION_ID: 'private_pushNotificationID',
```

Comment:
NAB: It isn't clear this ID is coming from Airship. Maybe a comment could help clarify.

Reply:
Still NAB, but bumping this comment

Reply:
Sorry, missed this. It is not coming from airship.

Reply:
Could we add a comment to explain where it is coming from?

Reply:
Done